generator/whitelist.ts

```diff
@@ -314,6 +314,16 @@ const whitelist: WhitelistConfig[] = [
         basePath: 'storagesync/resource-manager',
         namespace: 'Microsoft.StorageSync',
     },
+    {
+        basePath: 'softwareplan/resource-manager',
+        namespace: 'Microsoft.StorSimple',
+        resourceConfig: [
+            {
+                type: 'hybridUseBenefits',
+                scopes: ScopeType.ResourceGroup,
```

Comment:
This should be `ScopeType.Extension`. Please could you change this and re-run generation?